newt/builder/cmake.go

```diff
@@ -263,7 +244,7 @@ func CmakeCompilerInfoWrite(w io.Writer, archiveFile string, bpkg *BuildPackage,
 	includes = append(includes, c.GetLocalCompilerInfo().Includes...)
 	includes = append(includes, otherIncludes...)
 
-	includes = removeDuplicates(includes)
+	includes = util.SortFields(includes...)
```

Comment:
Is it not a problem to leave duplicates?

Reply:
This function also removes duplicates. It is used by newt when compiling, so it should be ok.

Reply:
Ah right, `SortFields` calls `UniqueStrings` but that is not documented behavior!